amundsen_application/static/js/components/common/ResourceListItem/DashboardListItem/index.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import * as Avatar from 'react-avatar';
+import { Link } from 'react-router-dom';
+
+import { LoggingParams } from '../types';
+
+import BookmarkIcon from 'components/common/Bookmark/BookmarkIcon';
+
+import { toTitleCase } from 'utils/stringUtils';
+
+export interface DashboardListItemProps {
+  dashboard: any; // TODO ttannis: Add DashboardResource type when we know what it is
```

Comment:
This is a type defined in `interfaces/Dashboard.ts` that's for a full-dashboard metadata. We could define a type `DashboardSummary` as a placeholder-- similarly I would like to create a `TableSummary` type to use in search results, `user/table/follows`, `user/table/reads`, `user/table/owns`, etc. This would also replace the `PopularTables` type as well.

Reply:
This makes sense. I'll update the code in this manner when the search endpoint is ready.